### PR TITLE
added a circleci job and updated the version constraint of symfony

### PR DESCRIPTION
**Pull request by drupalsyed**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -90,7 +95,7 @@ jobs:
           # fallback to using the latest cache if no exact match is found
           - v1-dependencies-
 
-      - run: composer install -n --prefer-dist
+      - run: composer update --prefer-lowest
```
Comment (deviantintegral) [inline .circleci/config.yml]: Something looks off here. This is inside the code style job, not the test job? Look at the composer commands that were actually run at:

* The [php-56-lowest](https://circleci.com/gh/turnerlabs/validating-xml-encoder/110?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) job
* The [code-fixer](https://circleci.com/gh/turnerlabs/validating-xml-encoder/106?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) job

**Review by deviantintegral — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (drupalsyed) [reply .circleci/config.yml]: Oh yes. I'll check on that.

File: .circleci/config.yml [hunk 2]
```diff
@@ -59,8 +59,38 @@ phpunit: &phpunit
       # run tests!
       - run: vendor/bin/phpunit
 
+phpunit1: &phpunit1
```
Comment (deviantintegral) [inline .circleci/config.yml]: Let's rename this to `phpunit-lowest`, and then feel free to merge if the jobs all pass. Nice work!

**Review by deviantintegral — APPROVED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (drupalsyed) [reply .circleci/config.yml]: Thanks @deviantintegral 👍  Yup ill rename it and merge when all the jobs pass.

**Pull request merged**